dpctl-capi/cmake/modules/FindLevelZero.cmake

```diff
@@ -25,8 +25,16 @@
 # LEVEL_ZERO_LIBRARY - the full path to the ze_loader library
 # TODO: Add a way to record the version of the level_zero library
 
+if(WIN32)
+    set L0_INCLUDE_DIR="%CD%\\level-zero\\include\\zet_api.h"
+endif()
+
 find_library(LEVEL_ZERO_LIBRARY ze_loader HINTS $ENV{L0_LIB_DIR})
-find_path(LEVEL_ZERO_INCLUDE_DIR NAMES level_zero/zet_api.h HINTS $ENV{L0_INCLUDE_DIR})
+if(UNIX)
+    find_path(LEVEL_ZERO_INCLUDE_DIR NAMES level_zero/zet_api.h HINTS $ENV{L0_INCLUDE_DIR})
+elif(WIN32)
+    find_path(LEVEL_ZERO_INCLUDE_DIR NAMES zet_api.h HINTS $ENV{L0_INCLUDE_DIR})
+endif()
```

Comment:
This change is not necessary if `L0_INCLUDE_DIR` is correct.

Reply:
It is that, because in source level-zero/include/zet_api.h, not level_zero/zet_api.h

Reply:
I think that build process copies `level-zero/include/zet_api.h` into `include/level_zero/zet_api.h`.